benefits/core/models.py

```diff
@@ -100,62 +100,6 @@ def __str__(self) -> str:
         return self.client_name
 
 
-class EligibilityType(models.Model):
-    """A single conditional eligibility type."""
-
-    id = models.AutoField(primary_key=True)
-    name = models.TextField()
-    label = models.TextField()
-    group_id = models.TextField()
-    supports_expiration = models.BooleanField(default=False)
-    expiration_days = models.PositiveSmallIntegerField(null=True, blank=True)
-    expiration_reenrollment_days = models.PositiveSmallIntegerField(null=True, blank=True)
-    enrollment_index_template = models.TextField(default="enrollment/index.html")
-    reenrollment_error_template = models.TextField(null=True, blank=True)
-    enrollment_success_template = models.TextField(default="enrollment/success.html")
-
-    def __str__(self):
-        return self.label
```

Comment:
Thanks, I made this change in the `EnrollmentFlow` model so that the `label` field will be used as the Admin display field for `EnrollmentFlow` instances.